### PR TITLE
Add redis configuration

### PR DESCRIPTION
**Pull request by tarnawski**

### CODE REVIEW

File: docker/redis/redis.conf [hunk 1]
```diff
@@ -0,0 +1,1316 @@
...
+#
+# A reasonable value for this option is 300 seconds, which is the new
+# Redis default starting with Redis 3.2.1.
+tcp-keepalive 300
+
+################################# GENERAL #####################################
+
+# By default Redis does not run as a daemon. Use 'yes' if you need it.
+# Note that Redis will write a pid file in /var/run/redis.pid when daemonized.
+daemonize no
```
Comment (dominikhajduk) [inline docker/redis/redis.conf:136]: set `yes` to be aple to run as deamon

File: docker/redis/redis.conf [hunk 2]
```diff
@@ -0,0 +1,1316 @@
...
+# debug (a lot of information, useful for development/testing)
+# verbose (many rarely useful info, but not a mess like the debug level)
+# notice (moderately verbose, what you want in production probably)
+# warning (only very important / critical messages are logged)
+loglevel notice
+
+# Specify the log file name. Also the empty string can be used to force
+# Redis to log on the standard output. Note that if you use standard
+# output for logging but daemonize, logs will be sent to /dev/null
+logfile ""
```
Comment (dominikhajduk) [inline docker/redis/redis.conf:171]: file to keep logs usually sth like `/var/log/redis_6379.log`

File: docker/redis/redis.conf [hunk 3]
```diff
@@ -0,0 +1,1316 @@
...
+
+# The working directory.
+#
+# The DB will be written inside this directory, with the filename specified
+# above using the 'dbfilename' configuration directive.
+#
+# The Append Only File will also be created inside this directory.
+#
+# Note that you must specify a directory here, not a file name.
+dir ./
```
Comment (dominikhajduk) [inline docker/redis/redis.conf:263]: place where redis will save snapshots

File: docker/redis/redis.conf [hunk 4]
```diff
@@ -0,0 +1,1316 @@
...
+#
+#   Note: you can disable saving completely by commenting out all "save" lines.
+#
+#   It is also possible to remove all the previously configured save
+#   points by adding a save directive with a single empty string argument
+#   like in the following example:
+#
+#   save ""
+
+save 900 1
```
Comment (dominikhajduk) [inline docker/redis/redis.conf:218]: config how often snapshot on disc will be saved

File: docker/redis/redis.conf [hunk 5]
```diff
@@ -0,0 +1,1316 @@
...
+#
+# By default protected mode is enabled. You should disable it only if
+# you are sure you want clients from other hosts to connect to Redis
+# even if no authentication is configured, nor a specific set of interfaces
+# are explicitly listed using the "bind" directive.
+protected-mode yes
+
+# Accept connections on the specified port, default is 6379 (IANA #815344).
+# If port 0 is specified Redis will not listen on a TCP socket.
+port 6379
```
Comment (dominikhajduk) [inline docker/redis/redis.conf:92]: port, usually not changed

File: docker/redis/redis.conf [hunk 6]
```diff
@@ -0,0 +1,1316 @@
...
+#    a given number of slaves.
+# 2) Redis slaves are able to perform a partial resynchronization with the
+#    master if the replication link is lost for a relatively small amount of
+#    time. You may want to configure the replication backlog size (see the next
+#    sections of this file) with a sensible value depending on your needs.
+# 3) Replication is automatic and does not need user intervention. After a
+#    network partition slaves automatically try to reconnect to masters
+#    and resynchronize with them.
+#
+# slaveof <masterip> <masterport>
```
Comment (dgrochowski) [inline docker/redis/redis.conf:281]: master - slave database replication

File: docker/redis/redis.conf [hunk 7]
```diff
@@ -0,0 +1,1316 @@
...
+# If you run Redis from upstart or systemd, Redis can interact with your
+# supervision tree. Options:
+#   supervised no      - no supervision interaction
+#   supervised upstart - signal upstart by putting Redis into SIGSTOP mode
+#   supervised systemd - signal systemd by writing READY=1 to $NOTIFY_SOCKET
+#   supervised auto    - detect upstart or systemd method based on
+#                        UPSTART_JOB or NOTIFY_SOCKET environment variables
+# Note: these supervision methods only signal "process is ready."
+#       They do not enable continuous liveness pings back to your supervisor.
+supervised no
```
Comment (dgrochowski) [inline docker/redis/redis.conf:147]: supervisor using to hold running redis service

File: docker/redis/redis.conf [hunk 8]
```diff
@@ -0,0 +1,1316 @@
...
+#
+# Specify the path for the Unix socket that will be used to listen for
+# incoming connections. There is no default, so Redis will not listen
+# on a unix socket when not specified.
+#
+# unixsocket /tmp/redis.sock
+# unixsocketperm 700
+
+# Close the connection after a client is idle for N seconds (0 to disable)
+timeout 0
```
Comment (dgrochowski) [inline docker/redis/redis.conf:113]: close client connection after x seconds

**Comment by tarnawski** [comment]
@dominikhajduk @dgrochowski Thanks for the comments.

**Pull request merged**